dbt_worker/tasks.py

```diff
@@ -97,7 +111,8 @@ def _invoke(task: Any, command: List[str], callback_url: Optional[str] = None):
     task_id = task.request.id
     _insert_log_path(command, task_id)
     logger.info(f"Running dbt task ({task_id}) with {command}")
-    # TODO: Send callback to infer task start.
+    if callback_url:
+        _send_state_callback(callback_url, task_id, STARTED)
```

Comment:
So at this point the task would technically show `PENDING` if we called for the status, right? I see how this gives the caller more accurate information, I'm just curious as to your thinking around sending this here rather than doing a `_get_task_status()` after we call `t.start()`. Is there a chance we tell a client the task has started and then it actually fails to start?

Reply:
Actually it shows STARTED not PENDING. PENDING = task is created buy still wait for worker process to pickup. STARTED = task has been picked up by worker. STARTED is automatically set by celery as I passed
 track_started=True. 

But you gave me a very good hint! I should try catch in callback helper to make sure it doesn't halt worker.

Reply:
Oh cool, thanks so much!